### PR TITLE
Extended the Action Plugins section of the scheduler in the official documentation

### PR DESCRIPTION
**Pull request by renhuanyu**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
 docs update
* **What this PR does / why we need it**:
Extended the Action Plugins section of the scheduler in the official documentation
* **Which issue(s) this PR fixes**:
#1285 Volcano reading materials improvement 
1.Make introductions for all Actions and Plugins both in Chinese and English, including how it works and best practices.
2.Demo about Northward computing framework on volcano.(Mpi/mindspore/tensorflow/kubeflow/flink/paddlepaddle)
3.Common scenarios and configurations about Scheduler.
4.Add a statu DAG in document about podgroup.
5.Add document about how to join volcano slack.

### CODE REVIEW

File: content/en/docs/schduler_introduction.md [hunk 1]
```diff
@@ -33,7 +34,7 @@ Volcano scheduler works as follows:
 
 ## Actions
 ### enqueue
-The enqueue action is responsible for filtering out the tasks that meet the scheduling requirements based on a series of filtering algorithms and sending them to the to-be-scheduled queue. After the action is executed, the status of the task changes from `pending` to `inqueue`. 
+The enqueue action is responsible for filtering out the tasks that meet the scheduling requirements based on a series of filtering algorithms and sending them to the to-be-scheduled queue. After the action is executed, the status of the task changes from `pending` to `inqueue`. mytest
```
Comment (huone1) [inline content/en/docs/schduler_introduction.md:37]: delete "mytest"

File: content/en/docs/actions.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+type = "docs"  # Do not modify.
+
+# Add menu entry to sidebar.
+linktitle = "Actions"
+[menu.docs]
+  parent = "scheduler"
+  weight = 2
++++
+
+### Enqueue
```
Comment (Thor-wl) [inline content/en/docs/actions.md:19]: delete pseudo code

File: content/en/docs/actions.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+  if node Adequate resources
+  inqueue := true;
+  if inqueue
+     enqueue
+  Queues.push(queue)
+} 
+
+```
+
+#### Logic
```
Comment (Thor-wl) [inline content/en/docs/actions.md:59]: %s/Logic/Overview

File: content/en/docs/actions.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+
+Filter the tasks that meet the requirements and enter the queue to be scheduled.The filtering condition is whether node meets the task's minimum resource requirement minReq.
+
+####  Scence
```
Comment (Thor-wl) [inline content/en/docs/actions.md:63]: %s/Scene/Scenario

File: content/zh/docs/actions.md [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+# Add menu entry to sidebar.
+linktitle = "Actions"
+[menu.docs]
+  parent = "scheduler"
+  weight = 2
++++
+
+### Enqueue
+
+```
```
Comment (Thor-wl) [inline content/zh/docs/actions.md:21]: delete pseudo code

File: content/zh/docs/actions.md [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+   if node资源足够 
+          inqueue := true;
+   if inqueue
+       enqueue
+   Queues.push(queue)
+} 
+
+```
+
+#### 逻辑
```
Comment (Thor-wl) [inline content/zh/docs/actions.md:56]: 简介

File: content/zh/docs/actions.md [hunk 7]
```diff
@@ -0,0 +1,223 @@
...
+
+筛选符合要求的任务进入待调度队列。过滤条件主要是对应任务再QueueMap是否存在，node是否满足任务的最小资源需求量minReq。
```
Comment (Thor-wl) [inline content/zh/docs/actions.md:58]: 筛选符合要求的“作业”，不是“任务”。在Volcano中“作业”（job）和“任务”（task）有严格定义，官方文档注意措辞严谨

File: content/zh/docs/actions.md [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+
+####  场景
+
+任务调度的准备过程，符合要求可以被调度的任务enqueue。任务状态由pending变为enqueue
```
Comment (Thor-wl) [inline content/zh/docs/actions.md:62]: “任务调度”更新为“作业调度”。官方文档避免口语化

File: content/zh/docs/actions.md [hunk 9]
```diff
@@ -0,0 +1,223 @@
...
+step2. pick a queue named Q from N (using ssn.QueueOrderFn)
+step3. pick a job named J from Q (using ssn.JobOrderFn)
+step4. pick a task T from J (using ssn.TaskOrderFn)
+step5. use predicateFn to filter out node that T can not be allocated on.
+step6. use ssn.NodeOrderFn to judge the best node and assign it to T
+```
+
+#### 逻辑
+
+<task,node>的绑定工作，包含预选和优选过程。使用predicateFn来过滤不能分配的node，使用NodeOrderFn打分来找到最好的node。
```
Comment (Thor-wl) [inline content/zh/docs/actions.md:81]: “最好的” -> "最适合的"

File: content/zh/docs/actions.md [hunk 10]
```diff
@@ -0,0 +1,223 @@
...
+
+#### 场景
+
+allocate从namespce层次开始遍历，不同的namespace可以代表不同业务的任务集合，这将有助于处理多类型的复杂业务场景的资源分配功能。不同的业务场景可以注册合适的调度算法(plugins中实现了多种具体的调度策略)
```
Comment (Thor-wl) [inline content/zh/docs/actions.md:85]: 这段表述优化一下，不体现逻辑，语义重点落到场景上

File: content/zh/docs/actions.md [hunk 11]
```diff
@@ -0,0 +1,223 @@
...
+若干抢占条件的过滤  
+//job内task的抢占
+For job in range underRequest
+若干抢占条件过滤
+
+```
+
+#### 逻辑
+
+这里抢占分为两个粒度，能够看到必须是同一个Queue下的job抢占，或者同一job下的task抢占。
```
Comment (Thor-wl) [inline content/zh/docs/actions.md:103]: Preempt用于同一个Queue中的Job之间的抢占，或同一Job下Task之间的抢占。

File: content/zh/docs/actions.md [hunk 12]
```diff
@@ -0,0 +1,223 @@
...
+
+#### 场景
+
+- Queue的粒度：相似的场景下发的任务进入到一个Queue中，多个Queue之间不存在资源的抢占。多个Queue之间对集群资源进行比例分配。在很多复杂的调度场景，按照业务对基本资源(cpu、磁盘、GPU、内存、网络带宽)的需求进行分类分组： 计算密集的场景如AI、高性能科学计算所对应的Queue的资源划分cpu、GPU、内存等计算资源需求高；spark框架等大数据场景磁盘需求高，等等。不同的Queue对资源的分配虽然是共享的，但如果AI场景抢占了所有的cpu资源，会导致别的场景对应的Queue中的任务饿死。因此分配基于Queue的粒度，就是为了保证资源的业务吞吐量。
```
Comment (Thor-wl) [inline content/zh/docs/actions.md:107]: 全篇所有的“作业”和“任务”用词检查更新一遍

**Comment by Thor-wl** [comment]
@renhuanyu Please add `--signoff` when push the commits

File: content/en/docs/actions.md (hunk 4 above)
Comment (Tingtal) [reply content/en/docs/actions.md]: Filter the tasks that meet the requirements and enter the queue to be scheduled. The filtering condition is whether the node meets the task's minimum resource requirement (minReq).

File: content/zh/docs/actions.md (hunk 9 above)
Comment (Tingtal) [reply content/zh/docs/actions.md]: Binding of <task,node>, including pre-selection and further selection. PredicateFn is used to filter out nodes that cannot be allocated, and NodeOrderFn is used to score the best node.

File: content/zh/docs/actions.md (hunk 11 above)
Comment (Tingtal) [reply content/zh/docs/actions.md]: The preempt action is used for resource preemption between jobs in a queue, or between tasks in a job.

File: content/zh/docs/actions.md (hunk 12 above)
Comment (Tingtal) [reply content/zh/docs/actions.md]: - Queue-based resource allocation: Jobs delivered in similar scenarios are placed in a queue, and resources are not preempted among multiple queues. Cluster resources are allocated to multiple queues in proportion. In complex scheduling scenarios, basic resources (CPUs, disks, GPUs, memory, network bandwidth) are allocated based on services: In computing-intensive scenarios, such as AI and high-performance scientific computing, queues require more computing resources, such as CPUs, GPUs, and memory. Big data scenarios, such as the Spark framework, have high requirements on disks. Different queues share resources. If AI jobs preempts all CPU resources, jobs in queues of other scenarios will starve. Therefore, the queue-based resource allocation is used to ensure service running.

File: content/zh/docs/plugins.md [hunk 13]
```diff
@@ -0,0 +1,223 @@
...
+
+根据业务的需要用户可以在自己的集群定制SLA相关参数。例如实时性服务要求较高的集群，JobWaitingTime可以设置的尽量小。批量计算作业为主的集群，JobWaitingTime可以设置较大。具体SLA的参数以及参数的优化需要结合具体的业务以及相关的性能测评结果。
+
+
+
+### Tdm
+
+#### 简介
+
+Tdm的全称是Time Division Multiplexing。在一些场景中，一些节点既属于Kubernetes集群的也属于Yarn集群的。Tdm plugin会标记这些节点为`revocable node`。Tdm plugin会在该类节点可被撤销的时间段内尝试把`preemptable task`调度给`revocable node`，并在该时间段内清除`revocable node`上的`preemptable task`。Tdm plugin提高了volcano在调度过程中节点资源的分时复用能力。
```
Comment (wpeng102) [inline content/zh/docs/plugins.md:189]: “一些节点既属于Kubernetes集群`的`也属于Yarn集群`的`” 删除`的`。
”Tdm plugin会标记这些节点为`revocable node`。”改为“Tdm plugin 需要管理员为这些节点标记为`revocable node`。”
“并在该时间段内清除”改为“并在该时间段之外清除”

File: config/_default/languages.toml [hunk 14]
```diff
@@ -82,18 +82,25 @@
   identifier = "concepts"
 
 [[zh.menu.docs]]
-  name = "Scheduler"
+  name = "生态"
   weight = 4
+  identifier = "zoology"
```
Comment (william-wang) [inline config/_default/languages.toml:87]: change 'zoology' to 'ecosystem'

File: content/en/docs/actions.md [hunk 15]
```diff
@@ -0,0 +1,117 @@
...
+
+Reclaim action tried to evict Pods from queues whose resource usage had exceeded the quota, and to allocate resources to queues whose resource usage had not been met. Reclaim action ensured the flexibility of cluster resource allocation and prevented the shock phenomenon of mutual expulsion among queues.
+
+
+
+### Elect
```
Comment (william-wang) [inline content/en/docs/actions.md:74]: The action has been deprecated from v1.2.  Please remove this action or mark it as deprecated.

File: content/en/docs/actions.md [hunk 16]
```diff
@@ -0,0 +1,117 @@
...
+
+- Job condition: v1.1.0 selects the job with the highest priority and the longest waiting time as the target job. In this way, not only can urgent task scenarios be prioritized, but jobs with more resource requirements are selected by default due to the consideration of waiting time length.
+
+- Number of jobs: The target job can be a single job or a group job. Considering the inevitable impact of resource reservation on scheduler performance in terms of throughput and latency, V1.1.0 uses a single target job approach.
+
+- Identification mode: Identification mode can be automatic identification or custom configuration. Currently, only automatic identification mode is supported, in which the scheduler automatically identifies the target jobs that meet the conditions and number and reserves resources for them during each scheduling cycle. Subsequent releases will consider supporting custom configuration at both global and queue granularity.
+
+  
+
+### Reserve
```
Comment (william-wang) [inline content/en/docs/actions.md:90]: The action has been deprecated from v1.2 and replaced with SLA plugin Please remove this action or mark it as deprecated.

File: content/zh/docs/plugins.md [hunk 17]
```diff
@@ -0,0 +1,223 @@
...
+# Add menu entry to sidebar.
+linktitle = "Plugins"
+[menu.docs]
+  parent = "scheduler"
+  weight = 3
++++
+
+### Gang
+
+{{<figure library="1" src="gang.png" title="Gang plugin">}}
```
Comment (william-wang) [inline content/zh/docs/plugins.md:20]: please change the gang png file to a new one.  It is not clear and official.

File: content/zh/docs/plugins.md [hunk 18]
```diff
@@ -0,0 +1,223 @@
...
+
+基于容器组概念的Gang调度算法十分适合需要多进程协作的场景。AI场景往往包含复杂的流程，Data Ingestion、Data Analysts、Data Splitting、Trainer、Serving、Logging等，需要一组容器进行协同工作，就很适合基于容器组的Gang调度策略。MPI计算框架下的多线程并行计算通信场景，由于需要主从进程协同工作，也非常适合使用Gang调度策略。容器组下的容器高度相关也可能存在资源争抢，整体调度分配，能够有效解决死锁。
+
+在集群资源不足的场景下，gang的调度策略对于集群资源的利用率的提升是非常明显的。
+
+
+
+### Binpack
+
+{{<figure library="1" src="binpack.png" title="binpack plugin">}}
```
Comment (william-wang) [inline content/zh/docs/plugins.md:36]: please change the binpack png file to a new one.  It is not clear and official.

File: content/zh/docs/plugins.md [hunk 19]
```diff
@@ -0,0 +1,223 @@
...
+#### 场景
+
+当集群运行了多个Job，但资源不足，并且每个Job下有不等数量的Pod等待被调度的时候，如果使用Kubernetes默认调度器，那么最终，具有更多Pod数量的Job将分得更多的集群资源。在这种情况下，volcano-scheduler提供算法支持不同的Job以fair-share的形式共享集群资源。
+
+Priority plugin能够让用户自定义job、task优先级，根据自己的需求在不同层次来定制调度策略。根据job的priorityClassName在应用层面进行优先级排序，例如集群中有金融场景、物联网监控场景等需要较高实时性的应用，Priority plugin能够保证其优先得到调度。
+
+
+
+### DRF
+{{<figure library="1" src="drf.png" title="drf plugin">}}
```
Comment (william-wang) [inline content/zh/docs/plugins.md:66]: please change the gang png file to a new one.

File: content/zh/docs/plugins.md [hunk 20]
```diff
@@ -0,0 +1,223 @@
...
+
+#### 场景
+
+DRF调度算法优先考虑集群中业务的吞吐量，适用单次AI训练、单次大数据计算以及查询等批处理小业务场景。
+
+
+
+### Proportion
+
+{{<figure library="1" src="proportion.png" title="proportion plugin">}}
```
Comment (william-wang) [inline content/zh/docs/plugins.md:79]: please change the png file to a new one.  It is not clear and official.

File: content/zh/docs/plugins.md [hunk 21]
```diff
@@ -0,0 +1,223 @@
...
+
+Conformance plugin会筛选牺牲页，使命名空间是kube-system的作业不能被驱逐。其能确保系统的关键资源不会被强制回收使用。
+
+#### 场景
+
+Conformance plugin能够保护在kube-system命名空间下的业务，这些业务是整个集群正常运行的保障。用户也可以根据其提供的模版，用于保护一些特定命名空间下的业务，使其不被抢占。   
+
+
+
+### Reservation
```
Comment (william-wang) [inline content/zh/docs/plugins.md:145]: already deprecated from v1.2 and replaced with SLA plugin.

File: content/zh/docs/plugins.md [hunk 22]
```diff
@@ -0,0 +1,223 @@
...
+- 支持cpu资源的拓扑调度。
+- 支持pod级别的拓扑协议。
+
+#### 场景
+
+Numa-aware的常见场景是那些对cpu参数敏感\调度延迟敏感的计算密集型作业。如科学计算、视频解码、动漫动画渲染、大数据离线处理等具体场景。
+
+
+
+#### Overcommit
```
Comment (william-wang) [inline content/zh/docs/plugins.md:219]: This is not introduction about overcommit?

**Review by Thor-wl — APPROVED** [review]

**Pull request merged**